docusaurus/src/components/SvgComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const SvgComponent = ({ icon }) => {
+  return (
+    <span style={{display: "flex", alignItems: "center", fontSize: 14}}>
+      <svg width={28} height={28} style={{marginRight: 10}}>
+        <use xlinkHref={`/img/icons.svg#${icon}`} />
+      </svg>
+      {icon}
+    </span>
+  );
+};
```

Comment:
1. make this file in `.ts`
2. add types to your component + write in short return